tests/publiccloud/run_ltp.pm

```diff
@@ -129,9 +142,19 @@ sub run {
             set_var('LTP_COMMAND_EXCLUDE', $exclude);
         }
     }
+   
+    my $runltp_ng_repo = '';
+    my $runltp_ng_branch = '';
 
-    my $runltp_ng_repo = get_var("LTP_RUN_NG_REPO", "https://github.com/metan-ucw/runltp-ng.git");
-    my $runltp_ng_branch = get_var("LTP_RUN_NG_BRANCH", "master");
+    if ( $ltp_runtime =~ /^python/i ) {
```

Comment:
I would not touch this lines when you trigger test with `LTP_RUNTIME=python`  you may just add also `LTP_RUN_NG_REPO=https://github.com/acerv/runltp-ng.git` and `LTP_RUN_NG_BRANCH=ssh` , having logic which set defaults to variables based on other variables it is too much

Reply:
Please, is this way also acceptable? https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/258b96b13606dfaf9801f3d33f985b128bff7fa6/tests/publiccloud/run_ltp.pm#L146-L147

Reply:
I removed the switch from here, keeping the original code: as clarified, for repo different from default we'll set the parameters at clone_job run